.github/workflows/generate_clients.yaml

```diff
@@ -0,0 +1,29 @@
+name: SDK Client Generator
+
+on:
+  repository_dispatch:
+    types: openapi-spec-change
+  
+jobs:
+  generate_sdk_clients:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Use Node.js 14
+      uses: actions/setup-node@v2
+      with:
+        node-version: '14'        
+    - name: Use Python 
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.9'
+    - name: Delete old generated giles 
```

Comment:
Removing old giles? https://en.m.wikipedia.org/wiki/Giles_(surname)

Reply:
Old Giles needs to go 😂

Reply:
Great catch. The sdks that are there are not upto date. The poetry publish script runs and I can successfully create master sdk locally but I need to clean this up on Monday